conf/defaults.ini

```diff
@@ -1299,6 +1299,17 @@ loki_basic_auth_password =
 # ex.
 # mylabelkey = mylabelvalue
 
+[unified_alerting.state_history.annotations]
+# This section controls retention of annotations automatically created while evaluating alert rules
+# when alerting state history backend is configured to be annotations (a setting [unified_alerting.state_history].backend
```

Comment:
not sure if we need the bracket here (a setting ?

Reply:
I amended the text. Please check again.